base/applications/network/netstat/netstat.c

```diff
@@ -98,7 +98,7 @@ BOOL ParseCmdline(int argc, wchar_t* argv[])
                         // UNIMPLEMENTED.
                         ConPuts(StdErr, L"'b' option is FIXME (Accepted option though unimplemented feature).\n");
                         bDoShowProcName = TRUE;
-#if _WIN32_WINNT < 0x600
+#if (_WIN32_WINNT < _WIN32_WINNT_VISTA || TRUE) /* prefer the old behavior */
```

Comment:
This will break compatibility with scripts, once we bump the target.

Reply:
Link to some real-world-example or "it didn't happen"(TM). Currently 2 vs 1 votes for keeping the current state.

Reply:
If you want to use voting, better create a poll on Mattermost. Very few people look into PRs.

Reply:
Not needed to split the voting from the PR, we can vote at Hermes post https://github.com/reactos/reactos/pull/5377#discussion_r1248249573

Reply:
I'd prefer the newer behaviour as well.

Reply:
Ok, I do summarize the votings as follows:
People who think we should stick with the 2k3sp2 beahvior even after version bump:
JoachimHenze, hbelusca

People who think we should automatically toggle to Win7 behavior after version bump:
binarymaster, learn-more, Darkfire, GeoB99

2nd fraction won democratically and I will remove the "|| TRUE" and the comment.